synapseclient/__main__.py

```diff
@@ -212,7 +212,8 @@ def associate(args, syn):
 
 def copy(args,syn):
     mappings = synapseutils.copy(syn, args.id, args.destinationId, 
-                         copyWikiPage=args.skipCopyWiki, 
+                         copyWikiPage=args.skipCopyWiki,
+                         copyAnnotations=args.skipAnnotations,
                          excludeTypes=args.excludeTypes, 
```

Comment:
That is actually the case, I know this is confusing.  args.skipCopyWiki defaults to True, as we want copyWikiPage = True defaults.  

It doesnt make sense for someone to do:
```
synapse cp syn1234 --destinationId syn3342 --copyWiki
```
everytime they want to copy wikipages.  Therefore:
```
synapse cp syn1234 --destinationId syn3342 --skipCopyWiki
```

This is the same for `skipAnnotations`